src/features/signin/SignIn.js

```diff
@@ -0,0 +1,90 @@
+//A sign in page. Taken mostly from material-ui examples https://github.com/mui-org/material-ui/tree/master/docs/src/pages/getting-started/templates/sign-in
```

Comment:
I'm not sure if you should merge this file just yet since the feature isn't fully implemented yet. We need to work out default routing and sessioning first to be safe.